### DISCUSSION TITLE
leading-normal

### DISCUSSION THREAD

**Discussion by Dorious** [tailwindlabs/tailwindcss#9007] [Help]
Can somebody explain to me why `leading-normal` is not `line-height: normal` but a 1.5? how is this "normal"?

**Comment by wongjn** [comment]
I think numbers are preferred here due to what the [`normal` keyword means](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#normal) in `line-height`:

> Depends on the user agent. Desktop browsers (including Firefox) use a default value of roughly `**1.2**`, depending on the element's `font-family`.

The word "roughly" and "Depends on the user agent" in there doesn't inspire confidence. With a number, it would be more consistent across browsers:

> The used value is this unitless [`<number>`](https://developer.mozilla.org/en-US/docs/Web/CSS/number) multiplied by the element's own font size.

Furthermore `1.5` promotes better accessibility than the roughly equivalent `1.2` as per [WCAG 1.4.12 Text spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing):

> […] no loss of content or functionality occurs by setting […] Line height (line spacing) to at least 1.5 times the font size;
> […]
> The intent of this Success Criterion (SC) is to ensure that when people override author specified text spacing to improve their reading experience, content is still readable and operable.

This then is complimented by [conformance technique C21, Specifying line spacing in CSS](https://www.w3.org/WAI/WCAG21/Techniques/css/C21):

> Many people with cognitive disabilities have trouble tracking lines of text when a block of text is single spaced. Providing spacing between 1.5 to 2 allows them to start a new line more easily once they have finished the previous one.
